packages/flutter/lib/src/material/drawer.dart

```diff
@@ -279,7 +302,9 @@ class DrawerController extends StatefulWidget {
 
   /// The color to use for the scrim that obscures primary content while a drawer is open.
```

Comment:
This (existing) comment assume that the reader understands what's meant by "primary content". It would helpful to explain what we mean.

Reply:
Re-worded